src-electron/generator/helper-endpointconfig.js

```diff
@@ -627,13 +628,19 @@ function endpoint_attribute_min_max_list(options) {
       .forEach((tok) => {
         switch (tok) {
           case 'def':
-            defMinMaxItems.push(`(uint16_t)${defS}`)
+            defMinMaxItems.push(
+              `(${mm.isTypeSigned ? '' : 'u'}int${mm.typeSize * 8}_t)${defS}`
```

Comment:
I suspect this will completely break things, since the actual type those structs store is in fact uint16_t, no?

Reply:
Did not cross check that. Just fixed this based on the typecasting being wrong